release-team/README.md

```diff
@@ -188,6 +195,60 @@ For all unplanned or embargoed releases
 
 ---
 
+## Release Team Selection
+
+In addition to discharging the duties of their respective Release Team roles, the current Release Team is charged with selecting the team for the following release cycle.
+
+Team selection should happen transparently.
+
+In the 1.11 release cycle, we selected the Release Team Lead by quorum during a public Release Team meeting and additional roles with staffed via [an issue in k/sig-release](https://github.com/kubernetes/sig-release/issues/167) and a set of PRs, starting with [this one](https://github.com/kubernetes/sig-release/pull/168).
+
+Each Release Team role is responsible for staffing their respective role, with this order of fall-through in mind:
+- training and selecting a successor from the current pool of role shadows
+- training and selecting a successor from non-Release Team members
+- staffing the role themselves
+
+Ultimately, if none of these can be satisfied, responsibility falls to the future Release Team Lead and SIG Release to staff the roles.
+
+### Release Team Lead Selection
+
+The incoming Release Team Lead _MUST_ have participated on the Release Team for 2 or more release cycles, acting in a lead (non-shadow) capacity for at least one of those cycles.
+
+Release Team Leads can be staffed, with this order of fall-through in mind:
+- the current pool of Release Team Lead shadows
+- the current pool of Release Team members
+- former Release Team members
+
+_The new Release Team Lead should be agreed by quorum of the current Release Team members._
```

Comment:
More precise definition of the quorum is likely here.

Reply:
Suggestion:

The new Release Team Lead may be nominated by any SIG-Release member, and will be selected by vote at a regular SIG-Release meeting before the end of the prior release cycle.  In the event that SIG-Release is not holding regular meetings, the prior Release Team Lead may convene an emergency SIG-Release meeting to select the new Release Team Lead by simple majority vote.

Reply:
> The new Release Team Lead may be nominated by any SIG-Release member

I'm not sure if we have a clear definition of the "SIG-Release member".

My suggestions would be "by any Kubernetes community representative, who is a member of the [Kubernetes organization on GitHub](https://github.com/orgs/kubernetes/people)". It solves two questions here:
- Kubernetes release process is the project-wide, not SIG-wide activity.
- We have a formal list of Kubernetes org members on GitHub, we don't have one for SIG-Release (what is fine, because SIG-related activities do not set any formal barrier as org membership does).

We have to work on the voting/election process more precisely, but it's not a blocker for the current PR.

Reply:
Yes: vote to merge this, then hammer out selection/endorsement process at the next SIG-release meeting.

Reply:
I'll add my $0.02 in, just so it's on record...

The Release Team is a subproject of SIG Release. They should have the agency to make decisions regarding the subproject. Any tiebreaking / contentious resolutions would fallback to the SIG Release Chairs, but only if required.

That said, the selection process should happen within the Release Team by the current Release Team members. 

This solves the question of "What constitutes SIG Release membership?" in the context of the Release Team.

SIG members are free to come and go as they please, so I'm not sure we should strive for a formal definition of a SIG Release member, outside of what is defined according to the contributor ladder.